StkAutomation/Python/Interval File Modification/Edit Interval File.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jul  8 11:56:06 2020
+
+@author: Jackson A Artis
+"""
+
+# Change working directory
+import os
+
+# Needs to be the directory holding the interval file
+os.chdir(r'\Users\jartis\Desktop\Satellite_Constellation_Access');
```

Comment:
I suggest making this a generic directory such as C:\temp instead of your desktop